sklearn/cross_validation.py

```diff
@@ -13,27 +12,29 @@
 from __future__ import division
 
 import warnings
-from itertools import chain, combinations
-from math import ceil, floor, factorial
 import numbers
-import time
+from itertools import combinations
 from abc import ABCMeta, abstractmethod
 
 import numpy as np
 import scipy.sparse as sp
+import sklearn.model_selection as mod_sel
```

Comment:
You should use relative imports in modules (and absolute imports in tests), see http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines.

You may as well move this import after the `from scipy.misc import comb`. As far as I can see imports tend to be roughly ordered like this: stdlib, numpy, scipy, sklearn.